client/packages/common/src/ui/components/inputs/TextInput/NumericTextInput.test.tsx

```diff
@@ -87,9 +87,9 @@ describe('Test NumericTextInput component', () => {
     const input = getByRole('textbox');
 
     fireEvent.change(input, { target: { value: '1.11' } });
-    expect(input).toHaveValue('1.1');
+    expect(input).toHaveValue('1.11');
```

Comment:
We no longer limit input, will only re-format on blur, so this is the correct display now